opsdroid/message.py

```diff
@@ -17,6 +17,9 @@ class Message(NewMessage):
     Deprecated. Use ``opsdroid.events.Message`` instead.
     """
 
+    # Skip registering this class with Event
+    _no_register = True
+
```

Comment:
Why not register `Message` with `Event`?

Reply:
because this is the old fallback message not the actual `Message` which is in `events.py`.

Reply:
Oh this is the old one, my bad.